modules/nf-core/iqtree/tests/main.nf.test

```diff
@@ -23,7 +23,38 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert process.out.phylogeny }
+                { assert path(process.out.phylogeny.get(0).get(1)).exists() },
+                { assert path(process.out.mldist.get(0).get(1)).exists() },
```

Comment:
shouldn't the distances be determnistic, e.g. allow you to snapshot `process.out mldist`?